compose/rest/record.go

```diff
@@ -174,7 +174,7 @@ func (ctrl *Record) Create(ctx context.Context, r *request.RecordCreate) (interf
 			NamespaceID: r.NamespaceID,
 			ModuleID:    r.ModuleID,
 			Values:      r.Values,
-			Labels:      r.Labels,
+			Meta:        r.Labels,
```

Comment:
This should be `Meta: r.Meta`, no?
Since we still take in both the `labels` and `meta` parameters, you'll need to check both here.